### DISCUSSION TITLE
Truncate/shorten series title in series list table?

### DISCUSSION THREAD

**Discussion by christian-0s** [Flexget/Flexget#3172] [Q&A] [answered]
Hi, is there a way to truncate a very long series title when displaying the list table in a terminal? Because series list command fails with "Could not start manager: invalid width 0 (must be > 0)" if terminal is not large enough.
An option would be to set a shortened title as main series name and full title in alternate_name, but I was wondering if there is a better way.

**Comment by lamski** [comment]
Have you tried `flexget series show series_name` (enclose the series name in double quotes if it contains spaces)?

**Reply by christian-0s** [reply]
Yes I know that command, but it shows only info for one specified series. If I need to look at all the series and one of them has a very long name, what happens is that flexget show less columns to fit the table in the terminal, and eventually fails.
```
┌───────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────┬──────────────────┬───────────────┐
│ Name                                                                                                      │ Begin │ Last Encountered │ Identified By │
├───────────────────────────────────────────────────────────────────────────────────────────────────────────┼───────┼──────────────────┼───────────────┤
│ Some series                                                                                               │ -     │ 5                │ sequence      │
│ Some other series with a very loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong title │ -     │ 5                │ sequence      │
└───────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────┴──────────────────┴───────────────┘
```
I was wondering is there's an easy way to define a maximum length of series name, after which the visualized title is truncated in the table, like this for example:
```
┌─────────────────────────────────┬───────┬──────────────────┬───────────────┐
│ Name                            │ Begin │ Last Encountered │ Identified By │
├─────────────────────────────────┼───────┼──────────────────┼───────────────┤
│ Some series                     │ -     │ 5                │ sequence      │
│ Some other series with a very...│ -     │ 5                │ sequence      │
└─────────────────────────────────┴───────┴──────────────────┴───────────────┘

**Comment by gazpachoking** [comment, marked as answer]
Are you using the latest version? We switched out table drawing library recently, so some of the behavior has changed.

**Reply by christian-0s** [reply]
I was using FlexGet-3.1.137, just switched to 3.1.152 and I see that something similar has already been implemented. 
Nice, thanks.
